### DISCUSSION TITLE
Combining video and audio files on same page

### DISCUSSION THREAD

**Discussion by victoria0527** [jspsych/jsPsych#841] [Q&A] [answered]
Is it possible to run a video with no sound at the same time and on the same page as a separate audio on jspsych? If so, how would I go about doing that?

I am trying to create "McGurk" stimuli with several video & audio file combinations.

**Comment by jodeleeuw** [comment, marked as answer]
If you need the precise match between video and audio for McGurk-effect stimuli then I'd recommend combining the audio files with video footage into separate video files and using those. I think the synchronization between video and independent audio will be hard to get just right. Loose synchronization is certainly possible, but I don't think that's good enough for a McGurk task.
